docs/docs/reference/dropped-features/wildcard-init.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: doc-page
+title: "Dropped: wildcard initializer"
+---
+
+The syntax
+```scala
+  var x: A = _
+```
+that was used to indicate an uninitialized field has been dropped.
```

Comment:
```suggestion
that was used to indicate an uninitialized field, has been dropped.
```

Ideally I'd want to add a comma between "syntax" and "that" but the inline code block makes that awkward to communicate.  But, anyways, perhaps it's more clear that the syntax is dropped not the field.